### PR TITLE
Translate ruby releases (ko)

### PR DESCRIPTION
**Pull request by riseshia**
Included:

- Ruby 2.2.6 Released
- Ruby 2.3.2 Released
- Ruby 2.3.3 Released

Please review this @ruby/www-ruby-lang-org-i18n-ko 
and Could someone suggest better translation for `저번 주에 릴리스된 루비 2.3.2 버전을 회귀한 것입니다.`? XD

### CODE REVIEW

File: ko/news/_posts/2016-11-15-ruby-2-2-6-released.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+author: "usa"
+translator: "shia"
+date: 2016-11-15 12:00:00 +0000
+lang: ko
+---
+
+루비 2.2.6가 릴리스 되었습니다.
+
+이번 릴리스는 RubyGems를 위한 새 SSL 인증서를 포함하고 있습니다.
+그리고 이전 릴리스로부터 80개의 버그 수정을 포함합니다.
```
Comment (yous) [inline ko/news/_posts/2016-11-15-ruby-2-2-6-released.md]: How about '그리고 이전 릴리스의 버그 80개를 수정합니다'?

File: ko/news/_posts/2016-11-15-ruby-2-2-6-released.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+layout: news_post
+title: "루비 2.2.6 릴리스"
+author: "usa"
+translator: "shia"
+date: 2016-11-15 12:00:00 +0000
+lang: ko
+---
+
+루비 2.2.6가 릴리스 되었습니다.
```
Comment (yous) [inline ko/news/_posts/2016-11-15-ruby-2-2-6-released.md]: How about '2.2.6이'? Do you read '2.2.6' either in English or Korean?

File: ko/news/_posts/2016-11-15-ruby-2-3-2-released.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+layout: news_post
+title: "루비 2.3.2 릴리스"
+author: "nagachika"
+translator: "shia"
+date: 2016-11-15 12:00:00 +0000
+lang: ko
+---
+
+루비 2.3.2이 릴리스 되었습니다.
```
Comment (yous) [inline ko/news/_posts/2016-11-15-ruby-2-3-2-released.md]: How about '2.3.2가'?

File: ko/news/_posts/2016-11-15-ruby-2-3-2-released.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+2.3 안정 버전 시리즈의 TEENY 버전입니다.
+
+이 릴리스는 RubyGems 2.5.2와 갱신된 ssl 인증서를 포함합니다.
+또한 많은 버그 수정도 포함되었습니다.
```
Comment (yous) [inline ko/news/_posts/2016-11-15-ruby-2-3-2-released.md]: There is an empty line above this line in original post, do we need to add one?

File: ko/news/_posts/2016-11-21-ruby-2-3-3-released.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+author: "nagachika"
+translator: "shia"
+date: 2016-11-21 10:30:00 +0000
+lang: ko
+---
+
+루비 2.3.3이 릴리스 되었습니다.
+
+이 릴리스는 Refinement와 Module#prepend에 대한 버그 수정을 포함하고 있습니다.
+같은 클래스에서 Module#refine와 Module#prepend를 섞어서 사용하게 되면
```
Comment (yous) [inline ko/news/_posts/2016-11-21-ruby-2-3-3-released.md]: Module#refine와 → Module#refine과

File: ko/news/_posts/2016-11-21-ruby-2-3-3-released.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+기대하지 않은 NoMethodError가 발생할 수 있습니다.
+저번 주에 릴리스된 루비 2.3.2 버전을 회귀한 것입니다.
```
Comment (yous) [inline ko/news/_posts/2016-11-21-ruby-2-3-3-released.md]: I think this sentence means something like '이는 지난주에 릴리스된 루비 2.3.2 버전의 회귀입니다'.

File: ko/news/_posts/2016-11-21-ruby-2-3-3-released.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+더 자세한 설명은 [[Bug #12920]](https://bugs.ruby-lang.org/issues/12920)
```
Comment (yous) [inline ko/news/_posts/2016-11-21-ruby-2-3-3-released.md]: \[\[Bug `#12920`\]\](...) → \[\[Bug `#12920`\]\](...)을 보세요.

**Comment by riseshia** [comment]
Applied all reviews, Thanks @yous !

**Comment by marocchino** [comment]
GTG

**Pull request merged**